org.altaqwaa.Altaqwaa.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.altaqwaa.Altaqwaa
```

Comment:
there not a single mention for `altaqwaa.org` on the github or in the appstream file. Is this domain really under the control of the upstream developers?

Reply:
Yes, I made the appstream file myself because there wasn't one upstream (electron generates one based off of `package.json` when making flatpak builds.) The website is empty for now, but don't worry, we plan on making it a verified app (hopefully a website too lol)